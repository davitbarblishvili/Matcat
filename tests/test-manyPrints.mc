func main() int
{
    printStr("hello world");
    string s;
    s = " with more words";
    printStr(s)
    return 0;
   
}