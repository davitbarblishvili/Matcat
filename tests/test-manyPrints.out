hello world with more words